modules/accredited_representative_portal/app/serializers/accredited_representative_portal/power_of_attorney_request_resolution_serializer.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+module AccreditedRepresentativePortal
+  class PowerOfAttorneyRequestResolutionSerializer
```

Comment:
Follow the library's [guide](https://github.com/jsonapi-serializer/jsonapi-serializer?tab=readme-ov-file#specifying-a-relationship-serializer) for polymorphic relationship instead.